bcda/client/bluebutton.go

```diff
@@ -142,29 +140,29 @@ func (bbc *BlueButtonClient) getData(path string, params url.Values, jobID strin
 
 	req.URL.RawQuery = params.Encode()
 
-	addRequestHeaders(req, reqID)
-
-	go logRequest(req, jobID)
-	resp, err := bbc.httpClient.Do(req)
-	if resp != nil {
-		logResponse(req, resp, jobID)
-	}
-	if err != nil {
-		return "", err
-	}
+	queryID := uuid.NewRandom()
+	addRequestHeaders(req, queryID)
 
-	defer resp.Body.Close()
+	tryCount := 0
+	maxTries := utils.GetEnvInt("BB_REQUEST_MAX_TRIES", 3)
+	retryInterval := utils.GetEnvInt("BB_REQUEST_RETRY_INTERVAL_MS", 1000)
 
-	if resp.StatusCode >= 400 {
-		return "", errors.New(resp.Status)
-	}
+	for tryCount < maxTries {
+		tryCount++
+		if tryCount > 1 {
+			logger.Infof("Retrying Blue Button request %s in %d ms...", queryID, retryInterval)
```

Comment:
Would also be nice to know the number attempted so far in this log statement :)

Reply:
Added! 
![Screen Shot 2019-06-21 at 11 36 13 AM](https://user-images.githubusercontent.com/1923441/59934514-b7c8e380-9419-11e9-9b28-0e7dd8e21a2c.png)